reference/forms/types/textarea.rst

```diff
@@ -30,7 +30,7 @@ Renders a ``textarea`` HTML element.
 .. tip::
 
     If you prefer to use an **advanced WYSIWYG editor** instead of a plain
-    textarea, consider using the FOSCKEditorBundle community bundle. Read
+    text area, consider using the FOSCKEditorBundle community bundle. Read
```

Comment:
Maybe use `textarea` instead?

Reply:
I'd even go for this to be more explicit:

```
``<textarea>``
```